daonode/src/main/java/bisq/daoNode/util/StaticFileHandler.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * This file is part of Bisq.
+ *
+ * Bisq is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or (at
+ * your option) any later version.
+ *
+ * Bisq is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public
+ * License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with Bisq. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package bisq.daoNode.util;
+
+import java.net.URI;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+
+import java.util.Arrays;
+
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+import lombok.extern.slf4j.Slf4j;
+
+
+
+import com.sun.net.httpserver.Headers;
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+
+/**
+ * JDK Server needs handler for serving files, will change in JDK 18
+ * Currently this is only to serve the swagger-ui content to the client.
+ * So any call to this handler must begin with api/v1. We keep v1 in case
+ * we will have incompatible changes in the future.
+ * This handler is limited to html,css,json and javascript files.
+ */
+@Slf4j
+@RequiredArgsConstructor
```

Comment:
I'm looking at Bisq2 dev guidelines:  Use Lombok annotations for Getter/Setter/ToString/EqualsAndHashCode. Be cautious with using more advanced Lombok features.

`@RequiredArgsConstructor:  Generates a constructor with required arguments. Required arguments are final fields and fields with constraints such as @NonNull.`  I've never used this annotation so I didn't automatically know what it does.  Now that I know, I see why the `classLoader` field is not final.  
I would retract my comment 
https://github.com/bisq-network/bisq/pull/6228/commits/41f640bdce0fd41dde56fba61fc16c9f65bbdba9#r890312779
but I leave it there to show how this `@RequiredArgsConstructor` annotation can confuse...

Reply:
Internal link does not work unless I paste in a new browser window.
[The comment below line 58.](https://github.com/bisq-network/bisq/pull/6228/commits/41f640bdce0fd41dde56fba61fc16c9f65bbdba9#r890312779)

Reply:
Yes, I remove it. Not a fan of those intransparent stuff...